system_metrics_collector/test/topic_statistics_collector/test_received_message_period.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
```

Comment:
should 3rdparty go after stdlib includes?

Reply:
Not per ```ament_cpplint```:

```
 Found C system header after other header. Should be: test_utilities.h, c system, c++ system, other.  [build/include_order] [4]
```

Reply:
This definitely fits in the "other" category. Is the issue that it's interpreting the angle brackets as a C or C++ system include?